src/tactic/ui/table/sobject_detail_wdg.py

```diff
@@ -44,6 +44,13 @@ class SObjectDetailElementWdg(BaseTableElementWdg):
         'type': 'SelectWdg',
         'values': 'true|false',
         'order': 3
+    },
+    'hide_default_elements': {
```

Comment:
Could you please rename the kwarg to show_default_elements instead?. The convention with our other kwargs is "show_...", like "show_gear" or "show_collection_tool". It's just for consistency. You may have to change the logic accordingly as well